src/content/docs/ru/tutorials/add-content-collections.mdx

```diff
@@ -0,0 +1,389 @@
+---
+title: Учебное пособие - Расширение с помощью коллекций контента
+description: >-
+  Преобразуйте код учебника Создать блог с маршрутизации на основе файлов на коллекции контента.
+i18nReady: true
+---
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro';
+import Box from '~/components/tutorial/Box.astro';
+import MultipleChoice from '~/components/tutorial/MultipleChoice.astro';
+import PreCheck from '~/components/tutorial/PreCheck.astro';
+import Option from '~/components/tutorial/Option.astro';
+
+**Коллекции контента** - это мощный способ управления группами схожего контента, например, записями в блоге. Коллекции помогают упорядочить документы, проверить YAML frontmatter и обеспечить автоматическую безопасность типов TypeScript для всего контента (даже если вы сами не пишете на TypeScript).
+
+<PreCheck>
+  - Переместите папку с записями блога в `src/content/`
+  - Создайте схему для определения frontmatter записей блога
+  - Используйте `getCollection()` для получения контента и метаданных записей блога
+</PreCheck>
+
+## Предварительные условия
+
+Вам понадобится **существующий проект Astro с файлами Markdown или MDX в папке `src/pages/`**.
+
+В этом уроке используется готовый проект [Готовый проект учебника "Создание блога"](https://github.com/withastro/blog-tutorial-demo) для демонстрации конвертации блога в коллекции контента. Вы можете использовать эту кодовую базу локально или завершить руководство в браузере, [отредактировав код руководства по блогу в StackBlitz](https://stackblitz.com/github/withastro/blog-tutorial-demo/tree/complete?file=src%2Fpages%2Findex.astro).
+
+Вместо этого вы можете выполнить эти шаги в своем собственном проекте Astro, но вам нужно будет скорректировать инструкции для вашей кодовой базы.
+
+Мы рекомендуем использовать наш пример проекта для завершения этого краткого руководства. Затем вы сможете использовать полученные знания для создания коллекций контента в своем собственном проекте.
+
+## Код учебника "Создание блога"
+
+Из вводного руководства по [созданию блога](/en/tutorial/0-introduction/) вы узнали о [встроенной файловой маршрутизации](/ru/guides/routing/#static-routes) Astro: любой файл `.astro`, `.md` или `.mdx` в папке `src/pages/` автоматически становился страницей на вашем сайте.
```

Comment:
We need to update the links to start with '/ru' to prevent errors from the cron job link checker
```suggestion
Из вводного руководства по [созданию блога](/ru/tutorial/0-introduction/) вы узнали о [встроенной файловой маршрутизации](/ru/guides/routing/#static-routes) Astro: любой файл `.astro`, `.md` или `.mdx` в папке `src/pages/` автоматически становился страницей на вашем сайте.
```